packages/amplify_auth_plugin_interface/lib/src/SignUp/ResendSignUpCodeResult.dart

```diff
@@ -13,16 +13,13 @@
  * permissions and limitations under the License.
  */
 
-
 import '../types/AuthCodeDeliveryDetails.dart';
 
-class ResendSignUpCodeResult { 
-  late AuthCodeDeliveryDetails codeDeliveryDetails;
-  ResendSignUpCodeResult({required codeDeliveryDetails}) {
-    this.codeDeliveryDetails = AuthCodeDeliveryDetails(
-      attributeName: codeDeliveryDetails["attributeName"] ?? "",
-      deliveryMedium: codeDeliveryDetails["deliveryMedium"] ?? "",
-      destination: codeDeliveryDetails["destination"]?? ""
-    );
-  }
+class ResendSignUpCodeResult {
+  AuthCodeDeliveryDetails codeDeliveryDetails;
+  ResendSignUpCodeResult({required codeDeliveryDetails})
+      : this.codeDeliveryDetails = AuthCodeDeliveryDetails(
+            attributeName: codeDeliveryDetails["attributeName"] ?? "",
+            deliveryMedium: codeDeliveryDetails["deliveryMedium"] ?? "",
+            destination: codeDeliveryDetails["destination"] ?? "");
 }
```

Comment:
I assume it's not possible to assert the type of the incoming `codeDeliveryDetails` parameter, is that correct?

Would it be better to update the `AuthCodeDeliveryDetails` class to provide default values of `""` rather than having to do this check anytime we wanted to implement this pattern? Thoughts?

Reply:
That makes a lot more sense.  I updated the code according to your suggestion here.